integrationtest/src/test/resources/fullFeatureTest/pom.xml

```diff
@@ -93,14 +93,12 @@
             </activation>
             <dependencies>
                 <dependency>
-                    <groupId>jakarta.xml.bind</groupId>
-                    <artifactId>jakarta.xml.bind-api</artifactId>
-                    <version>2.3.2</version>
+                    <groupId>javax.xml.bind</groupId>
+                    <artifactId>jaxb-api</artifactId>
                 </dependency>
                 <dependency>
                     <groupId>org.glassfish.jaxb</groupId>
                     <artifactId>jaxb-runtime</artifactId>
-                    <version>2.3.2</version>
                 </dependency>
```

Comment:
I changed `jakarta.xml.bind:jakarta.xml.bind-api:2.3.2` to `javax.xml.bind:jaxb-api:2.3.1`, because tests for Java 11 were failing with the following error:

```
[ERROR] mapstruct/processor/src/test/java/org/mapstruct/ap/test/builtin/bean/JakartaJaxbElementListProperty.java:[10,24] package jakarta.xml.bind does not exist
```

This happened in `Building fullFeatureTest` phase. But the same applies to the `jaxbTest`.

As far as I could understand this happens because of Maven's [dependency mediation](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#transitive-dependencies):


> Dependency mediation - this determines what version of an artifact will be chosen when multiple versions are encountered as dependencies. Maven picks the "nearest definition". That is, it uses the version of the closest dependency to your project in the tree of dependencies. You can always guarantee a version by declaring it explicitly in your project's POM. Note that if two dependency versions are at the same depth in the dependency tree, the first declaration wins. 

When integration tests are run using Java 8, everything is fine and tests pass as `jdk-11-or-newer` profile doesn't apply, and hence **there is only one** `jakarta.xml.bind:jakarta.xml.bind-api` dependency defined.

When integration tests are run using Java 11, `jdk-11-or-newer` profile applies, so now **there are two** `jakarta.xml.bind:jakarta.xml.bind-api` dependencies with **two different versions** defined.

I also can't just replace the version of the dependency (`2.3.2` => `3.0.0`), because then there will be no `javax.xml.bind` packages on the classpath, and, for example, the `jaxbTest` will fail.

This is why I replaced `jakarta.xml.bind:jakarta.xml.bind-api` with `javax.xml.bind:jaxb-api`. This allowed me to have both `javax.xml.bind` and `jakarta.xml.bind` packages on the classpath at the same time for Java 11.

---

The reason for removing the version tag of the `org.glassfish.jaxb:jaxb-runtime` is the same as [here](https://github.com/mapstruct/mapstruct/pull/2994#discussion_r958513597).

Reply:
Makes sense. I don't know what we where even using the jakarta dependency here